data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1298 @@ mission "Unfettered Jump Drive Trading"
+					goto vault
+			`	"We needed to act discreetly, so we would have the time to check the status of the Jump Drives without being swarmed by opportunist pirates. More people would have only attracted more attention and been more targets for the vault's defenses against. As ye have probably noticed, we have not accessed this vault for quite some time. Some unfettered humans could have snuck in and sabotaged the Jump Drives, like they did with the turrets." You have a hunch that there could be more to all this than what he's admitting.`
+				goto ship
+			label vault
+			`	"Yes and no. It be not only a holographic generator, but also a set of jammers, meaning your ship would not be able to pinpoint the exact location and would have to land blindly. We also needed to make sure the Jump Drives were not sabotaged and that the vault defenses were not compromised; they could probably have taken yer ship down. Needless to say, even monkeys could rig the Jump Drives to blow when used. Checking that takes time. Besides, we want to be using this vault later, so may as well ensure its safety. That said, it seems we left it sit too long, it be too compromised for further use." You have a hunch that there could be more to all this than what he's admitting.`
+			choice
+				`	(Go back to the ship.)`
+					goto ship
+				`	"If this world is so wild, why put the Jump Drives here in the first place?"`
+			`	"The Unfettered be not the ones who placed these Jump Drives here..." He leaves it at that. The Hai doesn't seem to want to tell you much about this vault.`
+			
+			label ship
+			`	As soon as your ship arrives, you give the orders to collect and either transfer the jump drives into cargo or install them directly onto your ship if possible, so you may flee more swiftly. When your ship takes off, you notice more activity than on the way in...`
+			`	The Hai comes to speak to you on the bridge. "Batten down the hatches, matey! It took us too long to check and load the jump drives, and we seem to have attracted attention. I guess a lookout got away from the vault and contacted some friends..." Multiple alarms start going off, confirming his suspicions - hostile ships are launching and heading straight for you. You launch as soon as you can, rushing through the correct procedures. Meanwhile, the Unfettered captain joins you on the bridge, clearly used to being where the action is, invited or not.`
+				launch
+	on accept
+		"outfit" "Jump Drive" 4
+	# a side mission will spawn additional ships if you brought a JD with you before the start of this mission.
+	npc evade
+		government "Cerberus"
+		personality disables plunders vindictive
+		system "Almaaz"
+		fleet "Large Northern Pirates"
+		fleet "Small Northern Pirates" 2
+
+
+
+# SPOILERS: the tsunami was caused by the Alphas as a warning to listen to their demands (and also because the Unfettered allowed you to see them)
```

Comment:
Um, what?
Causing a tsunami is no trivial matter

Reply:
make a crevasse, bam
that said they could have just caused any other disaster

Reply:
honestly since they have access to Korath tech I think just saying its a disaster can work; Koraths destroyed entire planets
(also it could always be natural cause too, I'm just writing what I had in mind when writing this)